support.html

```diff
@@ -0,0 +1,74 @@
+---
+stylesheet: index
+---
+
+{% include values.inc %}
+
+<div id="holder">
+    <section class="grid-middle hero">
+        <div class="col-5_md-12">
+            <h1>Support</h1>
+            <p>
+                You can ask questions in our community forums in <a href="{{ slackLink }}">Slack</a> or <a href="{{ slackLink }}">GitHub Discussions</a>, both of which have many knowledgeable users.
```

Comment:
How about making this a bullet list? Something like this:
```
The Rook community is here to support you:
- Ask questions in our community forums in <a href="{{ slackLink }}">Slack</a> or <a href="{{ githubDiscussionsLink }}">GitHub Discussions</a>
- To request a feature or report a defect, <a href="{{ newIssueLink }}">open a GitHub Issue</a>
- To contribute to the project, see the <a href="{{ contributeLink }}">Contribution Guide</a>
- For commercial support check out the <a href="#support-companies">list of Rook-related support and services companies</a> below.
```

Note we need to define `githubDiscussionsLink`